### PR TITLE
Templates: Update template using Andada

### LINKED ISSUE #10829: Templates: Update template using Andada
## Task Description

This is a follow-up to #10819, e6dc2e95961688d3935a9849587ae50426b0c0f2, and #10828

I realized that the “Food & Stuff” template uses the “Andada” font which was superseded by “Andada Pro” a while back.

We should update the template accordingly. Since the font only slightly changed, it should be a relatively simple update. It can be even done using a data migration so that we don't cause "Missing font" dialog to appear for lots of users.

Template to change: https://github.com/GoogleForCreators/web-stories-wp/blob/4cbc5bc0759e01653fed9e5c33fed3d55b59bc62/packages/templates/src/raw/food-and-stuff/template.json

<table>
<tr>
	<td>Before
	<td>After
<tr>
	<td>before
	<td>after
</table>

### Acceptance Criteria

* [ ] Add a new data migration to replace Andada font with Andada Pro, including font metrics. (Right now the metrics seem to be missing from the template)
* [ ] Add tests for data migration
* [ ] Run new data migration using `npm run workflow:migrate`. This will automatically update the Food & Stuff template.
* [ ] Unskip test from #10830

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Context

Acceptance Criteria can be found in original ticket. https://github.com/GoogleForCreators/web-stories-wp/issues/10829

## Summary

Replaces the Andada Font in the Food & Stuff template with Andada Pro.

Also adds a data migration to update existing stories automatically.

## Testing Instructions

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Create a new story from the Food & Stuff template and ensure that everything looks fine
2. Text elements should be using the Andada Pro font

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #10829

### CODE REVIEW

File: packages/migration/src/migrations/v0040_andadaFontToAndadaPro.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    capH: 705,
+    yMin: -382,
+    yMax: 1068,
+    hAsc: 942,
+    hDes: -235,
+    lGap: 0,
+  },
+  id: 'Andada Pro',
+  name: 'Andada Pro',
+  value: 'Andada Pro',
```
Comment (swissspidy) [inline packages/migration/src/migrations/v0040_andadaFontToAndadaPro.js]: These are not valid keys on an element's font object.

Need to remove these lines & update tests.

File: packages/templates/src/raw/food-and-stuff/template.json [hunk 2]
```diff
@@ -375,12 +375,43 @@
...
+              "capH": 705,
+              "yMin": -382,
+              "yMax": 1068,
+              "hAsc": 942,
+              "hDes": -235,
+              "lGap": 0
+            },
+            "id": "Andada Pro",
+            "name": "Andada Pro",
+            "value": "Andada Pro"
```
Comment (swissspidy) [inline packages/templates/src/raw/food-and-stuff/template.json]: As per https://github.com/GoogleForCreators/web-stories-wp/pull/10877/files#r823979533, let's remove these everywhere

File: packages/migration/src/migrations/v0040_andadaFontToAndadaPro.js (hunk 1 above)
Comment (spacedmonkey) [reply packages/migration/src/migrations/v0040_andadaFontToAndadaPro.js]: That interesting 🤔 I got these values, but copying them from values saved to the story object. This has id, name and value all saved. See this example
Screenshot 2022-03-10 at 18 16 24

I guessing these values being saved to the story are incorrect. Should I create a ticket to remove them?

Comment (swissspidy) [reply packages/migration/src/migrations/v0040_andadaFontToAndadaPro.js]: Yes please 👍 

I suppose these get added by accident by the font picker. Something @timarney is trying to circumvent in his font check dialog PR as well.

Comment (spacedmonkey) [reply packages/migration/src/migrations/v0040_andadaFontToAndadaPro.js]: Breakout issue - https://github.com/GoogleForCreators/web-stories-wp/issues/10885

**Review by swissspidy — APPROVED** [review]

**Pull request merged**